custom/icds_reports/tests/reports/test_awc_reports.py

```diff
@@ -2499,10 +2499,12 @@ def test_awc_report_pregnant_first_record(self):
                 reversed_order=False,
                 awc_id='a15'
             )
+
             self.assertEqual(
                 len(data['data']),
-                1
+                2
```

Comment:
This is set to '2'. because from where data is being fetched there `.exclude(open_in_month==False)`
is used. And But definition is a case is not open in month it cannot be pregnant. And Earlier pregnant cases were 2 but because of exclusion above one was not coming. Since I corrected the conflict in test data hence this is the result of that.